### PR TITLE
Removal of sandbox-classic module and binary

### PR DESCRIPTION
**Pull request by skisel-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: release/util.bzl [hunk 1]
```diff
@@ -8,7 +8,7 @@ inputs = {
     "sdk_config": ":sdk-config.yaml.tmpl",
     "install_sh": ":install.sh",
     "install_bat": ":install.bat",
-    "sandbox_logback": "//ledger/sandbox-common:src/main/resources/logback.xml",
+    "sandbox_logback": "//ledger/sandbox-on-x:src/app/resources/logback.xml",
```
Comment (skisel-da) [inline release/util.bzl:11]: I am not sure I am doing the right thing here. Where exactly this file is used? I have deleted an original one from `sandbox-common`

**Review by simonmaxen-da — APPROVED** [review]
LGTM

**Comment by tudor-da** [comment]
Nice job! Can you run full compat CI job as well before merging? (e.g. add `run-full-compat: true` after changelog_end in the git commit message)

File: release/util.bzl (hunk 1 above)
Comment (garyverhaegen-da) [reply release/util.bzl:11]: I think it's fine to delete. All of the other `*-logback.xml` files under `$DAML_HOME/sdk/$VERSION/daml-sdk` are used either directly in the [Daml Assistant code](https://github.com/digital-asset/daml/tree/main/daml-assistant/daml-helper/src/DA/Daml/Helper) or the [sdk config](https://github.com/digital-asset/daml/blob/main/release/sdk-config.yaml.tmpl).

Comment (skisel-da) [reply release/util.bzl]: done

**Pull request merged**